static/main.js

```diff
@@ -222,20 +229,27 @@ form.addEventListener("submit", (e) => {  // Submit button press event
     stop_button.style.color = "red";
 
     socket.emit("send", dict)
+    init_info_1.innerHTML = '<div style="color:yellow;font-size:50px;font-family: terminal, sans-serif;justify-content: center;align-items: center;display: flex;">Processing started</div>'
 });
 
 
 form.addEventListener("reset", (e) => {  // Stop button press event
     e.preventDefault();
-
-    let stop_button = document.getElementById("stop_button");
-    stop_button.style.color = "black";
-    processing_in_progress = false;
-
-    socket.emit("stop");
-
     let console_element = document.getElementById("init_info");
-    console_element.innerHTML += "Processing stopped\n";
+    if (processing_in_progress) {
+        let init_info_1 = document.getElementById("init_info_1");
+        init_info_1.innerHTML = "";
+        console_element.innerHTML = '<div style="color:red;font-size:50px;">Processing stopped</div>';
+        socket.emit("stop");
+        let stop_button = document.getElementById("stop_button");
+        stop_button.style.color = "black";
+        processing_in_progress = false;
+    } else {
+        let init_info_1 = document.getElementById("init_info_1");
+        init_info_1.innerHTML = "";
+        console_element.innerHTML = '<div style="color:red;font-size:50px;">Error: you haven`t started yet';
+        processing_in_progress = false;
```

Comment:
Unnecessary. You are in an if branch where `processing_in_progress` is false, so you don't need to set it to false again